packages/backend/src/server/api/endpoints/notes/search.ts

```diff
@@ -50,37 +50,44 @@ export const meta = {
 	},
 
 	res: {
-		type: 'array',
-		optional: false, nullable: false,
+		type: "array",
+		optional: false,
+		nullable: false,
 		items: {
-			type: 'object',
-			optional: false, nullable: false,
-			ref: 'Note',
+			type: "object",
+			optional: false,
+			nullable: false,
+			ref: "Note",
 		},
 	},
 
-	errors: {
-	},
+	errors: {},
```

Comment:
Please do not include such unnecessary changes as they will interfere with the review. 🙏

Reply:
The same goes for `res` and other changes in this file where single quotes were replaced with double quotes.

Reply:
Sorry about that, the code formatter did that.